Library/Homebrew/dev-cmd/linkage.rb

```diff
@@ -84,6 +88,21 @@ def display_normal_output
       display_items "Possible undeclared dependencies", @undeclared_deps
     end
 
+    def display_reverse_output
+      return if @reverse_links.empty?
+      puts "Links to:"
+      sorted = @reverse_links.sort
+      sorted.each do |dylib, files|
+        next if dylib == "/usr/lib/libSystem.B.dylib"
+        puts dylib
+        files.each do |f|
+          unprefixed = f.sub "#{@keg.to_s}/", ""
+          puts "  #{unprefixed}"
+        end
+        puts unless dylib == sorted.last[0]
```

Comment:
I'm finding these empty lines annoying and the indentation is sufficient to see the individual parts of the output. (But that's just personal preference and I'm not feeling too strongly about this.)